.github/workflows/service-druid-build-test-upload.yml

```diff
@@ -102,6 +102,8 @@ jobs:
         if: steps.changed-py-files.outputs.any_changed == 'true'
         run: |
           docker run --entrypoint "" ${{ env.IMAGE_NAME }} black --target-version py310 -S --check /usr/src/app
+      - name: Set permissions to coverage.xml
+        run: docker exec -d main_container chmod 666 /usr/src/app/coverage.xml
```

Comment:
Does it really work? `coverage.xml` is created after test suite finishes its job.